### DISCUSSION TITLE
[Feature] Show only selected Folders in the mobile APP

### DISCUSSION THREAD

**Discussion by zerion87** [immich-app/immich#4885] [Feature Request]
### The feature

In the mobile app, all Folders with Pictures (like Screenshots or the Instagram folder) are shown. Give th chance to use and show only self taken pictures with the camera, like Google Photos.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by aviv926** [comment]
It is entirely your choice whether to see the folders or not.
To do this in the app you will click on the cloud icon then on backup albums
And select an album you want to back up to and remove everything else from the selection

When finished, you will be able to see in the timeline only what is already backed up on the server + the files from the folder

If you want to choose not to see files from the phone, these are only what are on the server, you can create an empty folder without content and select it in the album backup option and this will cause Immich to display only the files on the server only
